src/Marten/Events/Projections/IAggregation.cs

```diff
@@ -1,6 +1,6 @@
 namespace Marten.Events.Projections
 {
-    public interface IAggregation<TAggregate, TEvent>
+    public interface IAggregation<in TAggregate, in TEvent>
```

Comment:
Let's definitely not do this one. I don't even want to let this kind of thing crop in right now until we absolutely have to have it for some unforeseeable use case.